cmd/run.go

```diff
@@ -457,6 +457,8 @@ func executeRun(runFilePath string, apps []runfileconfig.App) (bool, error) {
 	setupShutdownNotify(sigCh)
 
 	runStates := make([]*runExec.RunExec, 0, len(apps))
+	// Create a process group id for all the apps and daprd processes started.
```

Comment:
Please explain why.

Reply:
Also does it not create the Process Group ID for the current process only?

Reply:
have put better a description now. All children inherits the parent process group ID by default, until changed by following two - 
1. `Setpgid`
2. `&syscall.SysProcAttr{Setpgid: true}`

We are using the 1st option to our use and setting a different process group ID for `dapr run -f`. This will make all the children to inherit the same process group.

option 2 can also be used to create a different process group ID but it is not useful in our case as we want to set the process group ID for the calling process.